specification/cognitiveservices/data-plane/ComputerVision/v1.0/ComputerVision.json

```diff
@@ -0,0 +1,1482 @@
+                        "headers": {
+                            "Operation-Location": {
+                                "description": "URL to query for status of the operation. The operation ID will expire in 48 hours. ",
+                                "type": "string"
+                            }
+                        }
+                    },
+                    "default": {
+                        "schema": {
+                            "$ref": "#/definitions/ComputerVisionError"
+                        }
+                    }
+                },
+                "x-ms-examples": {
+                    "Successful Domain Model analysis request": {
+                        "$ref": "./examples//SuccessfulRecognizeTextWithUrl.json"
+                    }
+                }
+            }
+        },
+        "/textOperations/{operationId}": {
+            "get": {
+                "description": "This interface is used for getting text operation result. The URL to this interface should be retrieved from 'Operation-Location' field returned from Recognize Text interface.",
+                "operationId": "GetTextOperationResult",
+                "parameters": [
+                    {
+                        "name": "operationId",
+                        "in": "path",
+                        "description": "Id of the text operation returned in the response of the 'Recognize Handwritten Text'",
+                        "required": true,
+                        "type": "string"
+                    },
+                    {
+                        "$ref": "../../Common/Parameters.json#/parameters/SubscriptionKey"
+                    }
+                ],
+                "produces": [
+                    "application/json"
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Returns the operation status.",
+                        "schema": {
+                            "$ref": "#/definitions/TextOperationResult"
+                        }
+                    },
+                    "default": {
+                        "schema": {
+                            "$ref": "#/definitions/ComputerVisionError"
+                        }
+                    }
+                },
+                "x-ms-examples": {
+                    "Successful Domain Model analysis request": {
+                        "$ref": "./examples//SuccessfulGetTextOperationResult.json"
+                    }
+                }
+            }
+        }
+    },
+    "x-ms-paths": {
+        "/analyze?overload=stream": {
+            "post": {
+                "description": "This operation extracts a rich set of visual features based on the image content.",
+                "operationId": "Analyze Image In Stream",
```

Comment:
`operationId` must be `groupname_methodname` -- 

`groupname` is used to create a class in most generated languages
`methodname` is used to generate a method name in most languages